airflow/utils/log/logging_mixin.py

```diff
@@ -210,7 +210,11 @@ def set_context(logger, value):
             if hasattr(handler, "set_context"):
                 from airflow.utils.log.file_task_handler import FileTaskHandler
 
-                flag = cast(FileTaskHandler, handler).set_context(value)
+                if is_trigger and getattr(handler, "supports_triggerer", None):
+                    flag = cast(FileTaskHandler, handler).set_context(value, is_trigger=is_trigger)
+                    logger.propagate = False
```

Comment:
It feels like setting this shouldn't be done here given we have `flag` and `MAINTAIN_PROPAGATE`? (This feels like it would be confusing to have the flag essentially ignored?)

Reply:
i think i might not need this anymore.... lemme see

Reply:
yup, it was a remnant from the heavier-handed approach, where we were setting context on logging mixin in triggerer job... but now we delegate the context setting to the triggerwrapperhandler which sets it on the handler instance directly, and propagation is irrelevant because it's on root logger.